### PR TITLE
Feature/shorthand commands

### PR DESCRIPTION
**Pull request by tyrellbain**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Code style update
- [ ] Refactor (refactoring or adding test which isn't a fix or add a feature)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Did you test your solution?**

- [x] I lightly tested it
- [ ] I deeply tested it
- [ ] I wrote tests around it (unit tests, integration tests, E2E tests)

## Problem Description
Add shorthand commands for all available commands

## Solution Description
Add shorthands for `--verbose`, `--debug`, `--init`, `--quiet`. Update the exisiting shorthand for `--version`

## Side Effects, Risks, Impact

N/A

### CODE REVIEW

**Comment by gm-santos** [comment]
I resolved the conflicts and made some vulnerabilities updates, I think this one is good to go

**Review by guillebarbozaj3 — APPROVED** [review]
Test run fine! lgtm 🔥 
adviser test

File: src/cli/options.js [hunk 1]
```diff
@@ -70,8 +74,8 @@ module.exports = optionator({
     },
     {
       option: 'version',
-      alias: 'v',
       type: 'Boolean',
+      alias: 'r',
```
Comment (guillebarbozaj3) [inline src/cli/options.js:78]: I'd rather keep 'v' for version, lets discuss this in channel!

**Review by guillebarbozaj3 — CHANGES_REQUESTED** [review]

File: src/cli/options.js (hunk 1 above)
Comment (gm-santos) [reply src/cli/options.js]: I agree, put `vb` to `verbose` and change `r` to `v` for version

Comment (gm-santos) [reply src/cli/options.js]: I founded a better shorthand for `verbose`, keep `v` for `version` and use `V` for `verbose`

**Review by jslim — APPROVED** [review]
lgtm ty

**Pull request merged**